extensions/notebook/package.nls.json

```diff
@@ -32,5 +32,6 @@
 	"title.saveJupyterBook": "Save Book",
 	"title.searchJupyterBook": "Search Book",
 	"title.SavedBooks": "Saved Books",
-	"title.UnsavedBooks": "Unsaved Books"
+	"title.UnsavedBooks": "Unsaved Books",
+	"title.PreviewLocalizedBook": "Get localized BDC troubleshooting book"
```

Comment:
I'd run this by PM or something - wording seems a little off. Something like `Get a localized version of the BDC troubleshooting book`